content/en/security/cloud_security_management/setup/agentless_scanning/azure_resource_manager.md

```diff
@@ -0,0 +1,103 @@
+
+<div class="alert alert-warning">Running Agentless scanners incurs additional costs. To optimize these costs while still ensuring reliable 12-hour scans, Datadog recommends setting up <a href="/security/cloud_security_management/setup/agentless_scanning/terraform/">Agentless Scanning with Terraform</a> as the default template.</div>
+
+## Enable Agentless Scanning
+
+{{< tabs >}}
+{{% tab "New Azure subscription" %}}
+
+### Set up the Datadog Azure integration
+
+Follow the instructions for setting up the [Datadog Azure integration][1].
+
+### Enable Agentless Scanning for your Azure subscriptions
+
+Complete the following steps to enable Agentless Scanning for your Azure subscriptions:
+
+#### Cloud Security Management Setup page
+
+1. On the [Cloud Security Management Setup][2] page, click **Cloud Integrations** > **Azure**.
+1. Locate the tenant ID of your subscription.
+1. **(Optional)** To enable detection of misconfigurations, toggle **Resource Scanning** to the on position.
+1. Expand the list of Azure subscriptions and locate the subscription where you want to deploy the Agentless scanner.
+1. Click the **Enable** button under **Vulnerability Scanning**.
+1. The **Vulnerability Scanning** dialog is displayed. Toggle **Vulnerability Scanning** to the on position.
+1. Under **How would you like to set up Agentless Scanning?**, select **Azure Resource Manager**.
+1. Click **Launch Azure Resource Manager** to be redirected to the Azure portal.
+
+#### Azure portal
+
+1. Log in to the Azure portal. The template creation form is displayed.
+1. Select the subscription and the resource group in which the Agentless scanners will be deployed. Datadog recommends that you deploy the Datadog Agentless Scanner in a dedicated resource group.
+1. In **Subscriptions to scan**, select all the subscriptions you want to scan.
+1. Enter your **Datadog API Key**, select your **Datadog Site**, and fill out the remainder of the form.
+1. Click on **Review + create**.
+
+[1]: /integrations/guide/azure-manual-setup/?tab=azurecli
+[2]: https://app.datadoghq.com/security/configuration/csm/setup
+
+{{% /tab %}}
+
+{{% tab "Existing Azure subscription" %}}
+
+### Cloud Security Management Setup page
+1. On the [Cloud Security Management Setup][1] page, click **Cloud Integrations** > **Azure**.
+1. Locate the tenant ID of your subscription.
+1. **(Optional)** To enable detection of misconfigurations, toggle **Resource Scanning** to the on position.
+1. Expand the list of Azure subscriptions and locate the subscription where you want to deploy the Agentless scanner.
+1. Click the **Enable** button under **Vulnerability Scanning**.
+1. The **Vulnerability Scanning** dialog is displayed. Toggle **Vulnerability Scanning** to the on position.
+1. Under **How would you like to set up Agentless Scanning?**, select **Azure Resource Manager**.
+1. Click **Launch Azure Resource Manager** to be redirected to the Azure portal.
+
+### Azure portal
```

Comment:
These 2 subtitles are of level 3 in the "Existing Azure subscription" tab and of level 4 in the "New Azure subscription" tab. The rendering can be improved.
Can we homogenize? and maybe create a common section to avoid duplication?

Reply:
Yeah, that makes sense. We can use shortcodes to reuse the same content on both tabs. I'll go ahead and push a commit to the PR.